### DISCUSSION TITLE
Let's engage and collaborate in projects together

### DISCUSSION THREAD

**Discussion by formidablae** [formidablae/formidablae#36] [Let's collaborate]
## 👋 Heyyy!
 Let's work together on this huge thing I, you, everyone is doing. Engage and collaborate together 💪.
 Comment below and join the collaboration server on Discord: https://discord.gg/7cMaQejnQU

**Comment by shegz101** [comment]
I just started learning react, I do frontend development. I look forward to building a project with you. Still thinking on what the project will be.

**Reply by formidablae** [reply]
Thank you for your comment @shegz101. Very kind of you!

React is so cool! Good job :)

I would be very pleased too, to work together on a project. Will think on some ideas and propose them to you!

**Comment by ChrisToxz** [comment]
Hey,
I'm working on an opensource self hosted video streaming platform written in PHP. (Unreleased yet).
Any collaboration help is welcome actually but mainly looking for a front-end developer for the next version.

If you are interested or want more info, let me know!

**Reply by shegz101** [reply]
@ChrisToxz I am interested, and I'm a frontend developer. Just started my react journey tho.

**Reply by ChrisToxz** [reply]
@shegz101 Please drop me an email or let me know your Discord, thanks!

**Reply by shegz101** [reply]
I just sent you a message on discord

**Comment by hassan1344** [comment]
Hey!
I am a Software Engineer, and I would like to collaborate with you on open-source projects related to Javascript and Python.
Please drop me a message! Here is my discord : M Hassan#5978
